addons/survey/views/survey_templates_statistics.xml

```diff
@@ -62,6 +62,7 @@
         <t t-set="comment_lines" t-value="question_data['comment_line_ids']"/>
         <t t-set="graph_data" t-value="question_data['graph_data']"/>
         <t t-set="table_data" t-value="question_data['table_data']"/>
+        <t t-set="question_has_image_answers" t-value="any(image.value_image_filename for image in question.suggested_answer_ids)"/>
```

Comment:
Hmm would be "answer.value_image_filename for answer in..." then, as we are looping on answers and not necessarily images.

Sorry if my suggestion mislead you :smile:

Reply:
yeah makes more sense :sweat_smile: I'll change it accordingly